test/libdevcore/external_dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_policy(SET CMP0015 NEW)
+
+aux_source_directory(. SRCS)
```

Comment:
This is not the best way of adding files to CMake. The best is to just list all files.